app/emails/applications.py

```diff
@@ -244,7 +248,7 @@ def send_revert_notification(application_id, message):
         reversion_message=message
     )
 
-    subject = "Digital Marketplace application changes requested"
+    subject = "Feedback on your application to join Digital Marketplace"
```

Comment:
"...join **the** Digital Marketplace" ?

Reply:
This is as designed (Lucy Halliday is the best contact for this). I think the best way to think of it is using "Digital Marketplace" like you would other product nouns, e.g. "eBay".